scripts/frontend-development/docker-compose.yaml

```diff
@@ -26,3 +44,15 @@ services:
         condition: service_healthy
     ports:
       - "8080:8080"
+
+  # This fakes and SMTP email server.  User registration emails can be found by going to
+  #   localhost:1080 and opening the emails listed.
+  maildev:
```

Comment:
is the mail server needed just for the custom auth or also for the discord auth workflow?

Reply:
Yes it's just for the custom auth.

I'll enhance the comment in the next PR.